packages/wonder-blocks-data/src/util/get-gql-request-id.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import type {GqlOperation, GqlContext} from "./gql-types.js";
+
+const toString = (valid: mixed): string => {
+    if (typeof valid === "string") {
+        return valid;
+    }
+    return JSON.stringify(valid) ?? "";
+};
+
+/**
+ * Get an identifier for a given request.
+ */
+export const getGqlRequestId = <TData, TVariables: {...}>(
+    operation: GqlOperation<TData, TVariables>,
+    variables: ?TVariables,
+    context: GqlContext,
+): string => {
+    // We add all the bits for this into an array and then join them with
+    // a chosen separator.
+    const parts = [];
+
+    // First, we push the context values.
+    const sortedContext = Object.keys(context)
```

Comment:
I think you could use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams to simplify this logic a bit. I think calling `new URLSearchParams(context)` would do it? (and same below)

Reply:
Oh it does. It's just `sort()` doesn't return anything. Cool. :)